README.md

```diff
@@ -16,7 +16,7 @@ git clone https://github.com/contentful/discovery-app-react
 Install dependencies
 
 ```shell
-npm install
+yarn || npm install
```

Comment:
this might be confusing for some user we should either have just one shell command using yarn and mention for npm user that they can replace `yarn` by `npm` or better have two separate instructions, one for using yarn and one for using npm. Option one might be better though